internal/exec/vendor_utils.go

```diff
@@ -0,0 +1,352 @@
+package exec
+
+import (
```

Comment:
Also, how do updates work for the vendor command. What if the component already exists ? I'd expect the command will overwrite the existing components

Reply:
it will
we will implement `vendor diff` commands to show diff